README.md

```diff
@@ -64,6 +64,43 @@ issue use the method `connectToAdvertisingDevice` to first scan for the device a
 ```
 Besides the normal connection parameters that are described at above this function also has 2 additional required parameters: `withService` and  `prescanDuration`. PreScanDuration is the amount of time the ble stack will scan for the device before it attempts to connect (if the device is found)
 
+### Disconnect from device
```

Comment:
This is not the way to document API. Please start with adding a doc-comment to [connectToDevice(⋯)](https://github.com/PhilipsHue/flutter_reactive_ble/blob/adbc381fcb115373de51fcb426596ff508f81a5e/lib/src/reactive_ble.dart#L273). Then—as you in fact did—consider extending the example app. Only if after all of this you still think it's not enough, consider extending the readme.